attestation-service/verifier/src/lib.rs

```diff
@@ -152,6 +166,21 @@ pub trait Verifier {
         expected_report_data: &ReportData,
         expected_init_data_hash: &InitDataHash,
     ) -> Result<TeeEvidenceParsedClaim>;
+
+    /// Generate the supplemental challenge
+    /// 
+    /// Some TEE like IBM SE need a `challenge` generated on verifier side
+    /// and pass it to attester side. This challenge is used by attester to
+    /// generate the evidence
+    ///
+    /// A optional `tee_parameters` comes from the attester side as the input.
+
```

Comment:
If we want to use inline comment, we could delete extra empty lines between `///` lines and function definitions.

Reply:
Fixed, thanks!